web-marketplace/src/lib/ecocredit/api.ts

```diff
@@ -772,7 +786,19 @@ export const queryClasses = async ({
   request,
 }: QueryClassesProps): Promise<QueryClassesResponse> => {
   try {
-    return await client.Classes(request);
```

Comment:
I don't think this `queryClasses` function is actually used
I guess it would be nice to use it now in our react-query `getClassesQuery` that is just using `client.Classes(request)` right now

Reply:
Cool, done: https://github.com/regen-network/regen-web/pull/2356/commits/194174f301b7c5d899fa890436bd8b76027853d1